dev-python/lark-parser/lark-parser-0.11.1.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python module to propose a modern general-purpose parsing library for Python"
+HOMEPAGE="https://github.com/lark-parser/lark"
```

Comment:
Looks like upstream has tests. Could you add them please?

[This](https://dev.gentoo.org/~mgorny/python-guide/distutils.html#enabling-tests) might be useful.

Reply:
tests require additional dependency Js2Py which is not presented in the packages.

Reply:
Let's add it then! ;)

Reply:
It's available in ::guru so maybe it should graduate :) I ran those tests while trying py3.9 compat (which passed on the previous version I might add, but did have to fetch a module). [bug #769596](https://bugs.gentoo.org/769596)

Reply:
> It's available in ::guru so maybe it should graduate :) I ran those tests while trying py3.9 compat (which passed on the previous version I might add, but did have to fetch a module). [bug #769596](https://bugs.gentoo.org/769596)

@thesamesam @evdenis @ionenwks 
https://github.com/gentoo/gentoo/pull/19498

Reply:
Beaten to it!